util/format/format.go

```diff
@@ -174,19 +181,27 @@ func (f *flatFormatter) Format(format string, args ...interface{}) (n int, errno
 }
 
 // OutputFormat output escape character with backslash
-func OutputFormat(s string) string {
-	replace := map[string]string{
-		"'":  "''",
-		"\a": "\\a",
-		"\f": "f",
-		"\n": "\\n",
-		"\r": "\\r",
-		"\v": "v",
-	}
+func OutputFormat(s string, truncate bool, repeat bool) string {
```

Comment:
What's the meaning of `truncate` and `repeat`? Add comment for this function argument.

Reply:
`truncate` indicate if we need to stop handle following character when we meet '\0'.
`repeat` indicate if we need to use '\\0' to padding after we meet '\0' in the middle.
I'll comment that.